playbooks/webapp/README.md

```diff
@@ -157,6 +157,25 @@ To delete the webapp:
   ```
 * **deploy_flask_app_force_init** (bool): Whether to drop existing tables and create new ones when deploying the webapp database. Default: `false`
 
+### webapp deployment in HA architecture
```

Comment:
In this deployment deploy_flask_app_workers_instance_type uses a t3.micro and cannot see where it is documented. It will be probably better to create a dedicated README for the high availability deployment especially is some default values from the webapp  deployment are not retained.

Reply:
I have referenced the other variables used for this playbook here - https://github.com/redhat-cop/cloud.aws_ops/pull/97/files#diff-9ac3987075e323c6dbca1ada8d6a37a9768cbc9f22ae5168b3d84c49556422e1R164. I can add a link.

Reply:
Regarding the usage of t3.micro , based on [this comment](https://github.com/redhat-cop/cloud.aws_ops/pull/85#discussion_r1320006191), I am retaining the default value - 't2.xlarge'. I will remove t3.micro from the playbooks and use it only for our testing.